packages/release-please/src/release-please.ts

```diff
@@ -84,37 +94,30 @@ function releaseTypeFromRepoLanguage(language: string | null): string {
   }
 }
 
-// creates or updates the evergreen release-please release PR.
-async function createReleasePR(
-  releaseType: string,
-  packageName: string,
-  repoUrl: string,
-  github: GitHubAPI,
-  releaseLabels?: string[],
-  bumpMinorPreMajor?: boolean,
-  snapshot?: boolean,
-  path?: string,
-  monorepoTags?: boolean
-) {
-  const buildOptions: BuildOptions = {
-    packageName,
-    repoUrl,
-    apiUrl: DEFAULT_API_URL,
-    octokitAPIs: {
-      octokit: (github as {}) as OctokitType,
-      graphql: github.graphql,
-      request: github.request,
-    },
-    bumpMinorPreMajor,
-    snapshot,
-    path,
-    monorepoTags,
-  };
-  if (releaseLabels) {
-    buildOptions.label = releaseLabels.join(',');
+function findBranchConfiguration(
+  branch: string,
+  config: ConfigurationOptions
+): BranchConfiguration | null {
+  // look at primaryBranch first
+  if (branch === config.primaryBranch) {
+    return {
+      ...config,
+      ...{branch},
+    };
   }
 
-  await Runner.runner(ReleasePRFactory.build(releaseType, buildOptions));
+  if (!config.branches) {
```

Comment:
If no `primaryBranch` was set, but a config file existed, our behavior was to assume `master`, does this logic remain the same?

Reply:
In the additional branches configuration, `branch` is required.